scripts/Invoke-SasDeveloperWorkstation.py

```
@@ -0,0 +1,286 @@
+    else:
+        unsupported = False
+        inventory, inventory_step = run_inventory(platform, fixture, run_root, args.timeout_seconds)
+        steps.append(inventory_step); artifacts.append({"role": "inventory", "path": "inventory.json"})
+        selected_distro = "Ubuntu" if fixture and domain == "windows-wsl" else None
+        if inventory and domain == "windows-wsl" and not selected_distro:
+            selected = next((item for item in inventory.get("domains", []) if item.get("id") == "windows-wsl" and item.get("available") and not item.get("backend", {}).get("docker_only")), None)
+            selected_distro = selected.get("backend", {}).get("distribution") if selected else None
+        workspace_results = []
+        if args.mode != "Inventory":
+            plan_result, plan_step = run_workspace(platform, domain, "Plan", fixture, run_root, args.timeout_seconds, False, False)
+            steps.append(plan_step); artifacts.append({"role": "plan", "path": "workspace-plan.json"})
```

Ruff (0.15.21)

[error] Multiple statements on one line (semicolon)

(E702)